.github/workflows/firestore.yml

```diff
@@ -173,4 +173,5 @@ jobs:
       run: scripts/decrypt_gha_secret.sh scripts/gha-encrypted/FIREGSignInInfo.h.gpg \
           quickstart-ios/TestUtils/FIREGSignInInfo.h "$signin_secret"
     - name: Test quickstart
-      run: scripts/third_party/travis/retry.sh scripts/test_quickstart.sh Firestore
+      run: ([ -z $plist_secret ] || \
```

Comment:
Doesn't this have to be in the `env` for the task? As it stands this looks like this will never run `test_quickstart.sh`.

Reply:
See https://github.com/firebase/firebase-ios-sdk/runs/641984399?check_suite_focus=true for a working example for Storage.

Reply:
I see. Storage sets up `env` for the job instead of the specific steps. That change needs to be made also. Making that change and verifying here and in #5525. Thanks!